src/test/java/picard/illumina/parser/BinTdUtil.java

```diff
@@ -23,7 +23,7 @@ public static final String ltStr(final int lane, final int tile) {
     public static final byte C = (byte) 67;
     public static final byte G = (byte) 71;
     public static final byte T = (byte) 84;
-    public static final byte P = (byte) 46; //dot
+    public static final byte P = (byte) 'N';
```

Comment:
Shouldn't the variable now be named N also?
(and could you change the assignment for C to be C = (byte) 'C') - and the others too.

Reply:
And was there a particular reason you decided to get rid of '.' in favor of 'N' in this particular PR?

Reply:
The main reason was that decodeBasecalls had to translate this differently for fastqs vs sams which causes a performance hit. Really there is no reason not to use N everywhere.